xml/src/main/java/org/ehcache/xml/XmlConfiguration.java

```diff
@@ -402,8 +422,48 @@ public URL getURL() {
   @SuppressWarnings("unchecked")
   public <K, V> CacheConfigurationBuilder<K, V> newCacheConfigurationBuilderFromTemplate(final String name,
                                                                                          final Class<K> keyType,
-                                                                                         final Class<V> valueType)
+                                                                                         final Class<V> valueType,
+                                                                                         final ResourcePools resourcePools)
       throws InstantiationException, IllegalAccessException, ClassNotFoundException {
+    return internalCacheConfigurationBuilderFromTemplate(name, keyType, valueType, resourcePools);
+  }
+
+  /**
+   * Creates a new {@link CacheConfigurationBuilder} seeded with the cache-template configuration
```

Comment:
It does now, but the update was on next line so did not consider this comment as outdated.